### DISCUSSION TITLE
Slider Component

### DISCUSSION THREAD

**Discussion by Kaillera** [FakeFishGames/Barotrauma#4961] [Ideas and suggestions]
This may be just me but I want Slider components that looks similar to the slider from a battery/supercapaciter with adjustable values. I want both a vertical and horizontal sliders. This will allow players to quickly adjust values to their liking in their logic gates.

The slider would be 100 ticks wide. You can input two values, Low and High.

For instance setting Low as -1000 and High as 1000. You can move the slider and it'll send signals of the value. Putting the slider tick at say 70 will constantly send a signal of 400. Putting the slider tick at 0 will output -1000.

**Comment by mLuby** [comment]
Agreed.

In addition, it'd be great if the Slider were interact-able _without_ opening a battery-like GUI window. That is, the slider should show its position in-game like switches do. Clicking on one end of the slider could move the indicator one tick toward that direction, or even better would be click-dragging the indicator along the slider.
